### PR TITLE
Add port mapping in the setup documentation to make sure Jupyter notebook server port is exposed.

### PR DESCRIPTION
**Pull request by hsaputra**
Need to add `-p 8888:8888` to expose the port to host in the Docker documentation.

### CODE REVIEW

**Comment by ilblackdragon** [comment]
@tensorflow-jenkins test this please.

Need to pass checks even for documentation change.

**Comment by martinwicke** [comment]
Can you also add the flags to the docker_run script? And maybe add a sentence about what it's for?

**Comment by hsaputra** [comment]
This PR is for updating documentation.
Will send another PR to update the script. 

Would that be ok?

**Comment by martinwicke** [comment]
That would be ok. Can you add a sentence explaining why the -p argument is
necessary?

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ After Docker is installed, launch a Docker container with the TensorFlow binary
 image as follows.
```
Comment (ilblackdragon) [inline tensorflow/g3doc/get_started/os_setup.md]: how about adding here just simple to explain port, what it's for and that people can change it if they want to something else "(`-p 8888:8888` used to share internal port to the host machine, in this case to ensure to ensure Jupiter notebook connection)".

Comment (hsaputra) [inline tensorflow/g3doc/get_started/os_setup.md]: Sounds good. Will do

Comment (hsaputra) [inline tensorflow/g3doc/get_started/os_setup.md]: @martinwicke : thx, will do

**Comment by hsaputra** [comment]
Update PR with description about the Docker -p option.

**Comment by hsaputra** [comment]
Thx for merging, @martinwicke

**Pull request merged**